tests/itest_specs/basic_round_trip/long_call.cairo

```diff
@@ -70,13 +70,15 @@ namespace LongCallRoundTrip {
             contract_address=amm_addr,
             lptoken_address=lpt_call_addr
         );
-        assert call_pool_unlocked_capital_0 = 11529215046068469760; // 5ETH
+        assert call_pool_unlocked_capital_0.low = 5000000000000000000;
```

Comment:
out of curiosity... why didn't you keep the `assert_uint256_eq(call_pool_unlocked_capital_0, Uint256(5000000000000000000, 0));`?

Reply:
better error message, in this case it actually shows the value, otherwise it’s just assert 0 != 1